script/release/uploaders/upload.py

```diff
@@ -95,6 +97,20 @@ def main():
     shutil.copy2(os.path.join(OUT_DIR, 'debug.zip'), debug_zip)
     upload_electron(release, debug_zip, args)
 
+    # Upload libcxx_objects.zip for linux only
+    libcxx_objects = get_zip_name('libcxx_objects', ELECTRON_VERSION)
```

Comment:
```suggestion
    libcxx_objects = get_zip_name('libcxx-objects', ELECTRON_VERSION)
```

The release.js script uses a dash here